static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js

```diff
@@ -25,10 +25,12 @@ export const init = (): Promise<void> => {
         onConsentChange(state => {
             // Initial testing only
             console.log(getConsentFor('ipsos', state));
-            if(document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3") {
-                if (getConsentFor('ipsos', state)) {
-                    loadIpsosScript();
-                }
+            if(document.location.pathname === "/science/grrlscientist/2012/aug/07/3" || document.location.pathname === "/science/grrlscientist/2012/aug/07/3?dcr=true")
```

Comment:
`location.pathname` just includes the path, no query string, so you can remove the second URL check here I believe.

Reply:
Yep, you're right. Updated 👍